artifactory/auth/rtdetails.go

```diff
@@ -122,10 +123,14 @@ func (rt *artifactoryDetails) getArtifactoryVersion() (string, error) {
 	if resp.StatusCode != http.StatusOK {
 		return "", errorutils.CheckError(errors.New("Artifactory response: " + resp.Status + "\n" + utils.IndentJson(body)))
 	}
-
-	serverValues := strings.Split(resp.Header.Get("Server"), "/")
-	if len(serverValues) != 2 {
-		err = errors.New("Cannot parse Artifactory version from the server header.")
+	var version ArtifactoryVersion
+	err = json.Unmarshal(body, &version)
+	if err != nil {
+		return "", errorutils.CheckError(err)
 	}
-	return strings.TrimSpace(serverValues[1]), errorutils.CheckError(err)
+	return strings.TrimSpace(version.Version), nil
+}
+
+type ArtifactoryVersion struct {
```

Comment:
Can the *ArtifactoryVersion* type be made private and renamed to *artifactoryVersion*?

Reply:
Yes we can. I did the change and will do a PR